src/main/java/service/OpenWeatherAPI.java

```diff
@@ -0,0 +1,36 @@
+package service;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+public class OpenWeatherAPI implements WeatherDataSource {
+    @Override
+    public WeatherData getWeatherData() {
+        OkHttpClient client = new OkHttpClient().newBuilder().build();
+        Request request = new Request.Builder()
+                .url("https://api.open-meteo.com/v1/forecast?latitude=43.6532&longitude=-79.4163&" +
+                        "current=temperature_2m,precipitation&timezone=America%2FNew_York&forecast_days=1")
```

Comment:
You should change it to use the location data parameter after we merge my PR implementing the use case interactor